From 592d403a9b36d556e1c9a305ddc81a2477385cac Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Wed, 2 Dec 2020 10:42:52 +0100
Subject: [PATCH] Replace brotlipy with brotlicffi

The brotlipy package has been renamed to brotlicffi.  Update the imports
and dependencies accordingly.  The major advanage of the new package
is that it no longer collides with the Python bindings provided
by brotli itself.
---
 Pipfile            | 2 +-
 httpbin/filters.py | 2 +-
 setup.py           | 2 +-
 3 files changed, 3 insertions(+), 3 deletions(-)

#diff --git a/Pipfile b/Pipfile
#index 8ad29bb8..c63b526b 100644
#--- a/Pipfile
#+++ b/Pipfile
#@@ -5,7 +5,7 @@ verify_ssl = true
# [packages]
# gunicorn = "*"
# decorator = "*"
#-brotlipy = "*"
#+brotlicffi = "*"
# gevent = "*"
# Flask = "*"
# meinheld = "*"
diff --git a/httpbin/filters.py b/httpbin/filters.py
index c6268b6a..4deeaaad 100644
--- a/httpbin/filters.py
+++ b/httpbin/filters.py
@@ -10,7 +10,7 @@
 import gzip as gzip2
 import zlib
 
-import brotli as _brotli
+import brotlicffi as _brotli
 
 from six import BytesIO
 from decimal import Decimal
diff --git a/setup.py b/setup.py
index 32858e38..3f8700e2 100644
--- a/setup.py
+++ b/setup.py
@@ -34,7 +34,7 @@ setup(
     packages=find_packages(),
     include_package_data = True, # include files listed in MANIFEST.in
     install_requires=[
-        'Flask', 'MarkupSafe', 'decorator', 'itsdangerous', 'six', 'brotlipy',
+        'Flask', 'MarkupSafe', 'decorator', 'itsdangerous', 'six', 'brotlicffi',
         'raven[flask]', 'werkzeug>=0.14.1'
     ],
 )
